.bazelci/presubmit.yml

```diff
@@ -4,22 +4,48 @@ tasks:
   ubuntu1604:
     name: ":linux:"
     shell_commands:
-    - yes | ./configure
+    - wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+    - bash ./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda
+    - source $HOME/miniconda/bin/activate
+    - wget https://raw.githubusercontent.com/plaidml/plaidml/master/environment.yml
+    - $HOME/miniconda/bin/conda env update -f ./environment.yml
+    - conda activate $HOME/miniconda
     build_targets:
     - "//plaidml/keras:wheel"
+    build_options:
+    - "--config=linux_x86_64"
     test_targets:
     - "..."
+    test_options:
+    - "--cpu=linux_x86_64"
   macos:
     name: ":apple:"
     shell_commands:
-    - yes | ./configure
+    - curl -O https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh 
+    - sh ./Miniconda3-latest-MacOSX-x86_64.sh -b -p $HOME/miniconda
+    - curl -O https://raw.githubusercontent.com/plaidml/plaidml/master/environment.yml
+    - $HOME/miniconda/bin/conda env update -f ./environment.yml
+    - conda activate $HOME/miniconda
     build_targets:
     - "//plaidml/keras:wheel"
+    build_options:
+    - "--config=windows_x86_64"
     test_targets:
     - "..."
+    test_options:
+    - "--cpu=windows_x86_64"
   windows:
     name: ":window:"
+    batch_commands:
+    - curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe -OutFile .\miniconda.exe
+    - start /wait "" .\miniconda.exe /InstallationType=JustMe /S /D=%UserProfile%\Miniconda3
+    - curl https://raw.githubusercontent.com/plaidml/plaidml/master/environment-windows.yml -OutFile .\environment-windows.yml
+    - conda.exe env update -f .\environment-windows.yml
```

Comment:
We might have to deal with creating a conda environment vs updating one. It might be easier to have a python script that we call to handle all this.

Also, we need to somehow activate the environment before running the build on windows.

Reply:
Working on this now!